README.md

```diff
@@ -17,104 +17,139 @@ This documentation outlines the necessary and optional [dependencies](#dependenc
-```
+1. Once the mandatory user-provided dependencies are present, `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
 
-Alternatively, you can build **PURIFY** entirely from the source code. Once the mandatory dependencies are present, `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
+    ``` bash
+    git clone --recurse-submodules https://github.com/astro-informatics/purify.git
+    ```
 
-```
-git clone https://github.com/astro-informatics/purify.git
-```
+1. Create a `conan` package for `sopt`
 
-Then, the program can be built with standard `CMake` command:
+    ```bash
+    conan create /path/to/purify/sopt/ --build missing -s compiler.libcxx=libstdc++11 -pr:h=default -pr:b=default
+    ```
 
-```
-cd /path/to/code
-mkdir build
-cd build
-cmake -DCMAKE_BUILD_TYPE=Release ..
-make
-```
+1. Then, the program can be built using `conan`:
 
-To test everything went all right:
+    ``` bash
+    cd /path/to/purify
+    mkdir build
+	cd build
+    conan install .. --build missing -pr:h=default -pr:b=default
+    conan build ..
+    ```
 
-```
-cd /path/to/code/build
-ctest .
-```
+    You can turn the various options on and off by adding flags to the `conan install` command, e.g.
+	The full list of build options can be found in the [conanfile](./conanfile.py).
 
-To install in directory `/X`, with libraries going to `X/' do:
+    ```bash
+    conan install .. --build missing -o cppflow=on -o openmp=on -o mpi=off -pr:h=default -pr:b=default
+    ```
 
-```
-cd /path/to/code/build
-cmake -DCMAKE_INSTALL_PREFIX=/X ..
-make install
-```
+Installing and building PURIFY with TensorFlow
+-------
+
+The `sopt` library includes an interface to TensorFlow for using trained models as priors in the Forward-Backward optimization algorithm. To build **PURIFY** with TensorFlow capability, some extra steps are currently required. We aim to simplify the build process in a future release.
+
+1. Install the [TensorFlow C API](https://www.tensorflow.org/install/lang_c)
+1. Clone the UCL fork of `cppflow` and create a `conan` package using
+
+    ``` bash
+    git clone git@github.com:UCL/cppflow.git
+    conan create /path/to/cppflow/ -pr:h=default -pr:b=default
+    ```
+1. Once the mandatory user-provided dependencies are present, `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
+
+    ``` bash
+    git clone --recurse-submodules https://github.com/astro-informatics/purify.git
+    ```
+1. Create a `conan` package for `sopt` with the `cppflow` option set to "on"
+
+    ```bash
+    conan create /path/to/purify/sopt/ --build missing -s compiler.libcxx=libstdc++11 -o cppflow=on  -pr:h=default -pr:b=default
+    ```
+
+1. Then, the program can be built using `conan` with the `cppflow` option set to "on":
+
+    ``` bash
+    cd /path/to/purify
+    mkdir build
+    conan install .. --build missing -o cppflow=on -pr:h=default -pr:b=default
```

Comment:
```suggestion
    mkdir build
    cd build
    conan install .. --build missing -o cppflow=on -pr:h=default -pr:b=default
```

Reply:
Either need to be in the build folder or change the `..` to `.` in both the `conan install` and `conan build` commands